### DISCUSSION TITLE
Emadalshamery

### DISCUSSION THREAD

**Discussion by Emadalshamery** [community/community#140823] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

I need help 😔

**Comment by ebndev** [comment]
Hi @Emadalshamery, thanks for being a part of the GitHub Community!

You are more likely to get a useful response if you share more information about what you need help with. Giving a few more details might help someone give you a nudge in the right direction. 😄

**Reply by Emadalshamery** [reply]
Please help me i need some help with building network
